admin/taxonomy/class-taxonomy.php

```diff
@@ -342,50 +342,4 @@ private function insert_description_field_editor() {
 
 		add_action( "{$this->taxonomy}_term_edit_form_top", array( $this, 'custom_category_description_editor' ) );
 	}
-
-	/********************** DEPRECATED METHODS **********************/
```

Comment:
I'm not sure if we want to remove this header as we might have deprecated methods in the feature.